nodejs/restify/db.js

```diff
@@ -0,0 +1,40 @@
+const bcrypt     = require('bcrypt');
+const odbc       = require('odbc');
+const saltRounds = 12;
```

Comment:
Probably should put a comment with a link to https://www.npmjs.com/package/bcrypt#a-note-on-rounds